modules/apps/collaboration/wiki/wiki-service/src/main/java/com/liferay/wiki/service/impl/WikiPageServiceImpl.java

```diff
@@ -384,17 +384,25 @@ public String getNodePagesRSS(
 			feedURL, entryURL, attachmentURLPrefix, pages, false, null);
 	}
 
+	/**
+	 * @deprecated As of 2.0.0, replaced by {@link #getOrphans(WikiNode)}
+	 */
+	@Deprecated
 	@Override
 	public List<WikiPage> getOrphans(long groupId, long nodeId)
 		throws PortalException {
 
-		_wikiNodeModelResourcePermission.check(
-			getPermissionChecker(), nodeId, ActionKeys.VIEW);
+		WikiNode node = wikiNodeLocalService.getNode(nodeId);
+
+		return getOrphans(node);
+	}
 
-		List<WikiPage> pages = wikiPagePersistence.filterFindByG_N_H_S(
```

Comment:
Hey @Preston-Crary 
I think that this is a regression because since this is a remote service, we should only obtain the orphan pages from the set of pages the user is able to VIEW. We were doing this when we were invoking persistence.filterFindBy. However, with the new logic, we are just invoking the local service that doesn't do any permission checking because it executes:
```java
		List<WikiPage> pages = wikiPagePersistence.findByN_H_S(
			nodeId, true, WorkflowConstants.STATUS_APPROVED);
```

Could you please change this so we consider the permissions?
Thanks!